### DISCUSSION TITLE
Input Box Output

### DISCUSSION THREAD

**Discussion by rlandgraf2** [IBM/mac-ibm-notifications#33] [Q&A] [answered]
Been looking at this to replace some of our older message boxes that use pashua and jamfhelper.  i was looking to have to user enter a username in the input field but I do not see a way to pass that back out to the script that called the notification box.  I am wondering if I am missing something as it only seems to be passing out the value of the button pressed and not the value of the input box.  Any help with how to do this would be great.  Thanks

**Comment by SMartorelli** [comment, marked as answer]
Hi @rlandgraf2, Mac@IBM Notifications print the user input right before the exit of the process so you can easily catch it.
This is an example:
```
#!/bin/bash

# Required for executing a popup to the user space via an MDM depoloyment
CURRENTUSER=$(ls -l /dev/console | awk '{print $3}')

# Mac@IBM Notifications binary paths
NA_PATH="/Applications/Mac@IBM Notifications.app/Contents/MacOS/Mac@IBM Notifications"

# Variables for the popup notification for ease of customization
WINDOWTYPE="popup"

INPUT=$(sudo -u "${CURRENTUSER}" \
        "${NA_PATH}" \
        -type "${WINDOWTYPE}" \
        -title "Test input accessory view" \
        -accessory_view_type input \
        -accessory_view_payload "Placeholder")
        
echo "$INPUT"
```
